ocp4/profiles/moderate.profile

```diff
@@ -62,6 +62,7 @@ selections:
 
     ### Services
     # sshd
+    - service_sshd_disabled
```

Comment:
```suggestion
    #- service_sshd_disabled
```

Reply:
Looks like the build is failing by turning SSH off. We can comment out this rule, and then go through the ssh checks re-enabling/disabling as needed in new PRs.